doc/workflows/anaconda-webui.rst

```diff
@@ -0,0 +1,156 @@
+Anaconda WebUI Testing Workflow
+===============================
+
+Workflow for running Anaconda WebUI integration tests.
+
+Requirements
+------------
+
+- Testing framework, taken from anaconda, cockpit, cockpit-project/bots git repositories. Configured in settings.
+- Test scripts, taken from anaconda or other git repository. Configured in settings.
+- QEMU Virtualization hypervisor/s. Configured in settings.
+- Event structure InstallationSourceStructure, this is what is going to be tested.
+  It can be obtained by conversion from compose structure or supplied directly::
+  
+    {
+        "base_repo_id": "BaseOS",
+        "repos": {
+            "BaseOS": {
+                "x86_64": {
+                    "os": "http://example.com/compose/x86_64/BaseOS/os",
+                    "kernel": "images/pxeboot/vmlinuz",
+                    "initrd": "images/pxeboot/initrd.img"
+                }
+            }
+        }
+    }
+
+
+Settings
```

Comment:
Can you provide an example setting file that could be just copied and used? I for example do not know what is the format of the file.

Reply:
I've added link to the default settings file for the plugin, all the example should be in it.